src/docs/JobAnalytics/JobAnalyticsUserGuide.adoc

```diff
@@ -0,0 +1,96 @@
+[[_JA-duration]]
+
+image::JA-duration.png[align=center]
+- *Job variables chart:* This chart is intended to show all variable values of selected workflows. It helps to verify the workflow input values across all jobs. The chart provides the ability to hide or show specific input variables in the legend as shown in the figure below.
+
+- *Job results chart:* This chart is intended to show all result values of selected workflows. It helps to verify the workflow results across all jobs. The chart provides also the ability to hide or show specific results as shown in the figure above.
+
+image::JA-chart.png[align=center]
+
+All charts provides some advanced features such as "maximize" and "enlarge" to better view the results, and "move" to customize the dashboard layout (see top left side of charts). All of them provides the hovering feature as previously described and two types of charts to display: line and bar charts. Switching from one to another can be activated through a toggle button located in the top right of the chart. Same for show/hide variables and results.
+
+
+=== Job Execution table
+
+
+The last element of the Job Analytics dashboard shows a summary table that contains all job executions returned by the search query. It includes the job ID, status, duration, submission time, variables, results, etc.
+The summary table includes many features:
+
+- _Filtering_: users can specify filter values for every column. For instance, the picture below applies a filter on the duration where we filter only jobs that last more than 30s. For string values, we can apply string-related filters such as _contains_. For dates, a calendar is displayed to help users select the right date.
+- _Sort, hide, pin left and right columns_ to make users able to easily handle and display data according to their needs.
+- _Export the job data to another format_ to make users able to exploit and process them using other analytics tools such as R, Matlab, BI tools, Machine Learning APIs, etc.
+- _Clear and apply filters_: When filters are applied, the data might change so we provide an option to activate (see _apply filters to charts_ on the top left of the of the table screenshot) that allows to synchronize the charts with the filtered data in the table. Finally, it is possible to clear all filters. This will make the synchronization automatically deactivated.
+- _Link to scheduler jobs_: data in the job ID column is linked to the job executions in the scheduler. For example, if one wants to access to the logs of a failing job, he can click on the corresponding job ID to be redirected to the job location in the scheduling portal.
```

Comment:
he can click --> she/he can click
scheduling portal --> Scheduling Portal

Reply:
actually...
he can click -->  _they_ can click     ("they" is the inclusive pronoun for he/she).

also..
if one wants --> if the user wants to access

another option: keep "one"
if one wants ........... one can click on the corresponding...

Reply:
I opted for they ... they